Preparation for translatable jekyll content

maiska opened the pull request.

marmarek committed:
Add R4.1 release notes draft

marmarek commented:
Why adding `-documentation` to directories? It is already in qubes-doc repo, so it should be obvious, no? Is it to differentiate files (from non-documentation) in transifex?

gw0 committed:
Update resume-suspend-troubleshooting.md

Commit:
Fix SaltStack document links

Commit:
Use docs.saltproject.io for SaltStack document

Official SaltStack repository now uses saltproject.io, so use
new domain name to avoid unnecessary redirects.

example:
https://github.com/saltstack/salt/commit/dfa0619b9e55058b

Commit:
Improve wording

marmarek committed:
Remove extra newlines at the beginning/end of files

Those are redundant, and yaml parser strips them in fact. By removing
them, loading and saving yaml file without any change indeed produce the
same output. This is useful for prepare_for_translation.py script (which
adds lang and ref tags) - to produce only change that indeed was made.

marmarek committed:
Sort frontmatter entries

Again, this makes scripted modifications easier.

marmarek committed:
Fix core team link

marmarek committed:
Run script to add 'ref' parameters

Commit:
Merge branch 'patch-57' of https://github.com/adrelanos/qubes-doc into adrelanos-patch-57

Commit:
Wrap text

Commit:
Merge branch 'adrelanos-patch-57'

Commit:
Merge branch 'skip-autostart' of https://github.com/fepitre/qubes-doc into fepitre-skip-autostart

Commit:
Fix filename to conform to section naming scheme

Commit:
Update title and permalink to match filename

Commit:
Copyedit

Commit:
Merge branch 'fepitre-skip-autostart'

Commit:
Merge branch 'patch-17' of https://github.com/deeplow/qubes-doc into deeplow-patch-17

test90 committed:
merge remote

Commit:
Merge branch 'deeplow-patch-17'

test90 committed:
merge rmeote origin

ydirson committed:
qubes-builder: fix typo in rpmmacros filename, improve its markup

Commit:
Merge branch 'ydirson-rpmmacros-typo'

Commit:
Add guidelines regarding angled brackets around variables

Added in response to:
https://qubes-os.discourse.group/t/how-can-i-create-a-template-hvm-of-any-os/4676

Also updates terminology and clarifies wording.

Commit:
Add CI section; document `PipelineRetry` command (#1149)

Commit:
Document `TestDeploy`

Commit:
Update `TestDeploy` documentation

QubesOS/qubes-issues#6747

Commit:
Document qubes-secpack PGP key inclusion criteria

Commit:
Document wwwtest vs. wwwpreview

QubesOS/qubes-issues#6747

Commit:
Update Qubes Forum URL

Commit:
Update Qubes Forum URLs

ninavizz committed:
Update how-to-use-block-storage-devices.md

Remove use of the phrase "GUI" and specifically speak to the "Devices WIdget." Especially since it is now referenced in the Getting Started section, which should cross-link, here.

Commit:
Change name back to "Getting Started" and move to intro

QubesOS/qubes-issues#6756

Commit:
Change title back to "Getting Started"

QubesOS/qubes-issues#6756

Commit:
Update references to "Getting Started"

QubesOS/qubes-issues#6756

andrewdavidwong committed:
Fix capitalization

https://github.com/QubesOS/qubes-doc/pull/1169/files#r663353238

Commit:
Merge branch 'patch-2' of https://github.com/ninavizz/qubes-doc into ninavizz-patch-2

Commit:
Merge branch 'ninavizz-patch-2'

Commit:
Capitalize second word in title case compound adjectives

https://github.com/QubesOS/qubes-issues/issues/6756#issuecomment-873385039

Commit:
Capitalize second word in title case compound adjectives

https://github.com/QubesOS/qubes-issues/issues/6756#issuecomment-873385039

Commit:
Update "Help, Support, Mailing Lists, and Forum"

- Reorder sections (https://github.com/QubesOS/qubes-doc/pull/1170#issuecomment-874019640)
- Improve wording of "where to discuss Qubes" section (#1170)
- Add links to discussion guidelines

Commit:
Append periods to complete declarative sentence headings

Commit:
Use Markdown syntax instead of HTML

Although the HTMl tags are technically correct, it's probably more
important to preserve source readability and avoid the problems that
HTML-in-Markdown poses for localization and offline documentation.

Commit:
Update "Issue Tracking"

- Reorder sections (https://github.com/QubesOS/qubes-doc/pull/1170#issuecomment-874019640)
- Add explanatory text
- Improve wording
- Append periods to complete declarative sentence headings

Commit:
Update Whonix links; add Privacy Policy links

Commit:
Wrap text; improve wording

Commit:
Document template testing; improve related docs

- Document template testing repos
- Introduce dom0 testing repos
- Cross-link dom0 and domU testing repo sections
- Link to template testing section from related sections
- Add pointer to installing templates
- Minor improvements and fixes

Thanks to @GWeck for pointing out the need for this in:
https://forum.qubes-os.org/t/fedora-34-template-available-for-testing/4904/4

deeplow committed:
Minor improvements to the testbech instructions

deeplow committed:
add info on boot menu (part 1)

andrewdavidwong committed:
Use Markdown syntax instead of HTML

Although the HTML tags are technically correct, it's probably more
important to preserve source readability and avoid the problems that
HTML-in-Markdown poses for localization and offline documentation.

deeplow committed:
add steps to run script on every boot

deeplow committed:
start also script

Commit:
Revamp "Documentation Guidelines"

- Update contribution steps and first two images
- Improve source formatting and consistency
- Add information
- Improve organization and add subheadings
- Clarify and improve wording

Commit:
Merge branch 'patch-1' of https://github.com/gw0/qubes-doc into gw0-patch-1

Commit:
Merge branch 'gw0-patch-1'

Commit:
Merge branch 'patch-22' of https://github.com/deeplow/qubes-doc into deeplow-patch-22

Commit:
Merge branch 'deeplow-patch-22'

deeplow committed:
Update installation-guide.md

deeplow committed:
remove duplicated information about boot device

deeplow committed:
Replace all BIOS mentions with BIOS/UEFI

deeplow committed:
minor changes

dksmiffs committed:
Reduce Windows confusion in renamed 'Installation medium preparation' section

Commit:
Reorganize doc guidelines into multiple pages; update style guide

The existing doc guidelines page attempts to combine too many different
topics at once and includes information that does not pertain directly
to the documentation. This reorganization is intended to make each type
of information easier to find. For example, some have found it difficult
to find the documentation style guidelines (see, e.g.,
QubesOS/qubes-issues#6701#issuecomment-875875610). This reorganization
allows us to assign more specific titles to each page.

General changes:
- Create new page for contribution instructions
- Create new page for website style guide
- Create new page for continuous integration
- Rename existing "style guide" to "visual style guide" in order to
  avoid ambiguity with new doc and website style guides
- Retain existing page solely for doc style guide
- Update page names and permalinks
- Update existing links
- Improve language

Doc style guide changes:
- Add section on using sentence case in headings
  (see QubesOS/qubes-issues#6756 and #1173)
- Improve section organization
- Clarify language

In order to better preserve the Git history of each file, file renames
will be handled in a separate commit.

Commit:
Rename files in accordance with commit 676a13b

Commit:
Add note about QubesOS/qubes-issues#6769

Commit:
Improve section organization, formatting; fix link

Commit:
Use shorter anchor text for doc index link

unman committed:
Sentence case for titles

Commit:
Merge branch '6769' of https://github.com/unman/qubes-doc into unman-6769

Commit:
Fix capitalization; update terms (QubesOS/qubes-issues#6769)

Thank you to @unman for the initial sentence case pass.

Commit:
Remove note about QubesOS/qubes-issues#6769 (fixed)

Commit:
Merge branch 'unman-6769' (closes QubesOS/qubes-issues#6769)

Commit:
Merge branch 'patch-23' of https://github.com/deeplow/qubes-doc into deeplow-patch-23

Commit:
Fix and improve installation guide (#1172)

- Improve language
- Fix image links
- Explain BIOS and UEFI, provide links
- Fill in missing steps and information gaps
- Standardize grammar and punctuation
- Improve formatting
- Clarify instructions and explanations

Commit:
Fix typo; improve image titles (#1172)

Commit:
Merge branch 'deeplow-patch-23'

Commit:
Update doc style guide and instructions

- Explain why image linking is important
- Improve wording
- Fix link
- Minor improvements

Commit:
Update example to reflect current style guide

Commit:
Fix typo

Commit:
Add section on serving website locally

Commit:
Cross-link doc style guide and how-to pages

Commit:
Change "version" to "release"

See https://github.com/QubesOS/qubes-issues/issues/5308#issuecomment-877700342

Commit:
Update filename

Commit:
Fix typo

Commit:
Update titles and links

Commit:
Change "point relase" to "patch release" for precision

In general, the software world is not clear about these terms. However,
the most common usage seems to use "point release" to refer to any
release with at least one period in the version number, which would also
include minor releases. By contrast, "patch release," though a much less
common phrase, unambiguously denotes a release with a version number
containing two periods.

See:
https://en.wikipedia.org/wiki/Point_release
https://semver.org/

Commit:
Update patch release description for accuracy

Commit:
Update terminology

Commit:
Fix typo

dksmiffs committed:
Undo change to 'Copying the ISO' header, not worth the broken links

dksmiffs committed:
Merge tag 'adw_6eea2bf3'

Tag for commit 6eea2bf39456647a2ecbdde5344a213b7aad3c31

# -----BEGIN PGP SIGNATURE-----
#
# iQIzBAABCgAdFiEE4R0VxtIENXaf+qRWjOE3NSoBmhcFAmDqGVAACgkQjOE3NSoB
# mhcsdg/6A2zT+GvdR0XtnGxyJoI1WDTHrTSt9e0BiVOWorCs1OoL85leR0jvzWgO
# hcXeIxiQK7EnG/QKQC1UMY3gSfL+t8bkaGlXVqcNPFdbVIiHqg/w0KuMaA1pq4mQ
# slGtcZyQjjWLLm8RiL4KxshnqVZWqokQwOg3GwL72bNvVxOGb5nJAI9VIyq9+C6N
# rqE3qUg3CkzVUxjEFh8kNjGGc/SrpWRlXDHr/6+L5BgGKGcXjhy3S480yRNHU5LI
# V/xqkFa+QJaA5F0E+2CIX+fuHSQGl8HrupdXltBRT4VYTibLICGAPsiibpETSrNX
# f0lPXUfJAQINw9MAWEvypuFhCHc21xwV/KHVymGuUsbLVFsSUDFrAimb9zBxvqRC
# MNAnIBcwrvgxBBg2P4XVTLaV6fKe8vGf3fuiV0Rlxpjk/5oq8KKwCc9LeSd1Q+Sq
# hHV72j/yOpXe/FJ10eWMJQLoOGUk/aMu8gAhpjSMBzuDV9kmwp3H2eNNrTrU3hxW
# UAvSaNTCK+klmNTWfiPKDaQCtJFhrBqglzo/yNo0JUBhpP6azZ41ISU9pd9lamc5
# kbBWioRjeb6hQ+aBgU6W2UY+q9S6gyvITpfB+PmjU+HzCUqoxpKKe2epQnnm63Tn
# 46eceH/DdlFWZcRP0Zi2aMsdyylwRaGVoRASwb90JKRtfVSEd58=
# =gYJ6
# -----END PGP SIGNATURE-----
# gpg: Signature made 7/10/2021 5:04:00 PM Central Daylight Time
# gpg:                using RSA key E11D15C6D20435769FFAA4568CE137352A019A17
# gpg: Can't check signature: No public key

dksmiffs committed:
Add clarification based on PR #1174 feedback

dksmiffs committed:
Fix link to HTML syntax (#1174)

andrewdavidwong committed:
Copyedit

Commit:
Merge branch 'dksmiffs-master'

Commit:
Update title to distinguish from main HCL page

Moving redirects to main HCL page in a separate commit.

Commit:
Update filename to match new title

Commit:
Move files back to qubes-doc so they can appear in index

I'm still not sure why these files have to be in qubes-doc in order to
appear in the doc index, but they do, so I'm moving them back even
though they're mostly empty.

deeplow committed:
test-bench: section on installing testing deps

deeplow committed:
minor fixes

dksmiffs committed:
Fix htmlproofer errors in PRs #1176 & #1177...

caused by hcl.md file rename in qubes-doc commit 7d894a59.

Commit:
Merge branch 'dksmiffs-master'

dksmiffs committed:
Move 'properly validated keys' guidance before git verification...

...in the "How to Verify Qubes Repos" section, since you must have
properly validated keys before being able to perform a successful
`git verify-tag` or `git verify-commit`.

deeplow committed:
Fix captization & add links

deeplow committed:
fix typo

deeplow committed:
fix captizalization

dksmiffs committed:
Fix lost referent on "this verification" (#1179)

dksmiffs committed:
Add another possible reason for "WARNING: This key is not certified...

...with a trusted signature! There is no indication that the signature
belongs to the owner."

With this edit, I'm aiming to assist the beginner reader who walked the
following breadcrumbs:

Should I trust this website?
==> verify the PGP signatures on the commits and/or tags

Detailed steps suggested by the docs along these breadcrumbs:
1) `git clone git@github.com:QubesOS/qubesos.github.io.git`
2) Verify the PGP sigs on the commits and/or tags
  a) get properly validated GPG keys (available in the Qubes Security Pack)
    i.   `git clone https://github.com/QubesOS/qubes-secpack.git`
    ii.  `gpg --import qubes-secpack/keys/*/*`
    iii. Verify/trust the QMSK (details given on the page)
  b) `cd qubesos.github.io`
  c) `git verify-commit 45ca80e8` (one of Andrew's recent commits)

RESULT: The WARNING pops up, because the user has not yet ultimately PGP
  trusted Andrew's E11D 15C6 D204 3576 9FFA  A456 8CE1 3735 2A01 9A17
  key.

andrewdavidwong committed:
Add missing article

dksmiffs committed:
Resolve grammatical error, return to "this verification" wording (#1179)

deeplow committed:
add bash as code-block language

Commit:
Merge branch 'dksmiffs-master'

Commit:
Fix headings

See https://github.com/QubesOS/qubes-doc/pull/1180#issuecomment-883103461

Commit:
Merge branch 'master' of https://github.com/dksmiffs/qubes-doc into dksmiffs-master

Commit:
Merge branch 'dksmiffs-master'

Commit:
Revamp "Verifying signatures" and "Qubes security pack"

- Standardize syntax, formatting, and orthography
- Update links
- Clarify and improve language
- Deduplicate content
- Add info regarding keys not directly signed by the QMSK
  (https://github.com/QubesOS/qubes-doc/pull/1180#discussion_r672939404)
- Add general info about digital signature use
- Improve page organization (#1179)
- Update info and instructions regarding the QMSK
- Miscelanneous fixes and improvements

Commit:
Update links; fix typo

Commit:
Update "Verifying signatures" and "Qubes security pack"

- Improve language
- Improve organization
- Fix typos
- Clarify steps in instructions
- Improve formatting

Commit:
Merge branch 'verifying-sigs-secpack'

Commit:
Fix link

Commit:
Merge branch 'fix20210614' of https://github.com/zpc0/qubes-doc into zpc0-fix20210614

Commit:
Merge branch 'zpc0-fix20210614'

Commit:
Improve ordering, presentation, and organization of information

- Generalize "PGP key inclusion criteria" to "PGP key policies"
- Move key signing notice to new section
- Deduplicate key signing notice
- Convert bolded lines to headings

deeplow committed:
add extra case for installation freezing

If installing in external SSD sometimes and when the user forgets to disable sys-usb the initial setup also
fails after setup networking.

dksmiffs committed:
Avoid repeating the prereqs in the 'How to verify the [...] hash values' section...

...also make other minor improvements.

dksmiffs committed:
Correct spelling error that broke the first #1184 check

deeplow committed:
fix typo

Commit:
Merge branch 'deeplow-patch-25'

Commit:
Add sentence about joining the testing team

https://forum.qubes-os.org/t/testing-team-how-to-deal-with-in-forum/4453/49?u=adw

dksmiffs committed:
Resolve some of the #1184 requested changes.

Commit:
Merge branch 'master' of https://github.com/dksmiffs/qubes-doc into dksmiffs-master

Commit:
Start links at verbs to match link target content

Commit:
Merge branch 'dksmiffs-master'

Commit:
Attempt to document `PipelineRefresh`

See https://github.com/QubesOS/qubes-posts/pull/82#issuecomment-888231099

Commit:
Document 'zenity' requirement for GUI inter-qube copying

Fixes QubesOS/qubes-issues#6801

Commit:
Clarify scope with respect to official vs. unofficial venues

The need for this clarification became apparent in:
https://forum.qubes-os.org/t/comments-from-staff-core-team-etc-regarding-facebook-group/5433/

Commit:
Add "videos"

deeplow committed:
remove unneeded qubesd restart

andrewdavidwong committed:
Update developer/system/template-manager.md

Co-authored-by: Marta Marczykowska-Górecka <marmarta@users.noreply.github.com>

Commit:
Merge branch 'qvm-template' of https://github.com/WillyPillow/qubes-doc

Commit:
Revise in accordance with documentation style guide

SvenSemmler committed:
Add "Just works" for R4.0.4 and the CPU info

Link to the forum post with the community created "just works" list for R4.0.4; also added the respective CPU info the the testing hardware retrieved from @marmarek

Commit:
Merge branch 'SvenSemmler-patch-1'

Commit:
Add note about appropriate reports; improve organization

Commit:
Update hardware-related docs

- Add links to "just works" list
- Improve organization
- Standardize orthography
- Wrap text
- Remove very old ML link
- Minor wording improvements

Commit:
Merge branch 'security-report'

Commit:
Fix typo; standardize orthography

Commit:
Standardize orthography

Commit:
Merge branch 'hardware'

Commit:
Remove release number from URLs (#1187)

This won't break the URLs. They'll just automatically redirect to the
correct thread. This will allow us to keep updating and re-using the
same thread and URL without scaring off users who see "r4-0-4" in the
URL and think it's out-of-date in the future.

Commit:
Standardize orthography

Commit:
Standardize orthography

Commit:
Genericize anchor text and shorten URL (#1187)

Mitigates against future changes in thread title.

Commit:
Add FAQ entry about not fixing all upstream bugs

This topic seems to come up frequently enough to merit an FAQ entry. The
most recent example is https://github.com/QubesOS/qubes-issues/issues/6825#issuecomment-895266155,
but it is by no means the first.

Commit:
Add note about backports

Commit:
Update new issue instructions to account for simplified template

Commit:
Update milestone names; standardize orthography

Commit:
Remove specific Qubes version refs and warn against this mistake

Commit:
Minor clarifications

deeplow committed:
fix typo

Commit:
Merge branch 'deeplow-patch-26'

Commit:
Update Qubes OS 4.1 release notes

- Add issues bearing the "release notes" label on the 4.1
  milestone (incomplete items should be removed before finalizing)
- Improve language
- Standardize syntax and formatting

Commit:
Deprecate "Hardware testing" page

Convert it into a redirect to the community-recommended hardware list on
the Qubes Forum. File rename will be in subsequent commit to maintain
history. See:
https://forum.qubes-os.org/t/short-list-of-laptops-desktops-that-work-well-with-qubes-os/5197/159?u=adw

Commit:
Rename file to match new contents

See previous commit message. File rename is in this separate commit to
maintain file history in Git.

Commit:
Update "Certified hardware"

- Add requirement that certified devices must be available for purchase
  with Qubes OS preinstalled. See: https://forum.qubes-os.org/t/5795/
- Improve wording
- Add link to HCL in intro
- Wrap text

SvenSemmler committed:
add link to external documentation

I just copy & pasted this part of the page into an [answer on the forum](https://forum.qubes-os.org/t/6189/) and realized it would be nice if "External documentation" would also be hyperlinked. If there is a reason this was not done, please just ignore this PR.

Commit:
Add QubesOS/qubes-issues#6877 to release notes

Commit:
Fix link URL and title; wrap text

Commit:
Merge branch 'SvenSemmler-patch-3'

Commit:
Update intro to explain basic versioning and terminology

Commit:
Update "version scheme"

- Add info about major, minor, and patch releases
- Add info about common abbreviations
- Add links
- Improve language
- Improve formatting
- Fix typos
- Convert table from HTML to Markdown

Commit:
Add link to "bug priorities" section

Commit:
Update "version scheme"

- Minor clarification
- Improve formatting
- Fix and improve orthography
- Make image a link

Commit:
Change "netvm" to "net qube"

QubesOS/qubes-issues#1015

Commit:
Note exception for templates

Commit:
Clarify "Grub in UEFI mode"

Commit:
Fix link

3hhh committed:
Update device-handling-security.md

minor clarifications

Commit:
Merge branch '3hhh-patch-1'

SvenSemmler committed:
Changed security archive layout for bullseye

I submitted #1191 previously but then had already purged my local fork. So I was unable to change it (if there is a way I am oblivious to it). Hence I closed #1191 and opened this pull request in it's place. Sorry for that.

Commit:
Change mailing list posting policies due to excessive spam

Commit:
Merge branch 'patch-1' of https://github.com/SvenSemmler/qubes-doc into SvenSemmler-patch-1

Commit:
Improve languge, syntax, and formatting; add link; wrap text

Commit:
Merge branch 'SvenSemmler-patch-1'

Commit:
Add Whonix 16 support; clarify that upstream EOL supersedes table

Commit:
Merge branch 'qvm-template' of https://github.com/WillyPillow/qubes-doc

Commit:
Merge branch 'patch-24' of https://github.com/deeplow/qubes-doc into deeplow-patch-24

Commit:
Merge branch 'deeplow-patch-24'

deeplow committed:
fix md typo

Commit:
Merge branch 'deeplow-patch-27'

Commit:
Add link to "Upgrading to R4.1"

Commit:
Improve organization of upgrade guides

Commit:
Merge branch 'upgrade-reorg'

Commit:
Create initial 4.1 upgrade page

Commit:
Update link

marmarek committed:
Fill in R4.0->R4.1 upgrade steps

marmarek committed:
Update R4.1 release notes

marmarek committed:
Rename release notes file to match others

marmarek committed:
Add R4.1 release schedule

Commit:
Add links; improve language

Commit:
Merge branch 'pr-828'

Commit:
Add and update links; wrap text

Commit:
Merge branch 'upgrade-4.1'

Commit:
Update supported 4.1 templates

Commit:
Explain "major version number"

https://forum.qubes-os.org/t/7027/

Commit:
Fixes QubesOS/qubes-issues#7050

Commit:
Add F34 template support to 4.0

QubesOS/qubes-issues#6568

Ark-of-the-copepod committed:
wrong package name, line 237

For Debian, `ncat` works not `nmap-ncat`.

Commit:
Merge branch 'Ark-of-the-copepod-patch-1'

Commit:
Remove Whonix 15 support (EOL)

unman committed:
Review template use before trying to remove

unman committed:
State requirement for networking from minimal templates.

Commit:
Merge branch 'template'

Commit:
Merge branch 'min-net'

mvermaes committed:
Trivial typo - remove extraneous bracket

Commit:
Merge branch 'mvermaes-patch-1'

Commit:
AMD typo

correct multiple instances of ADM->AMD

Commit:
Merge branch 'crd477-crd477-patch-1'

runephilosof-abtion committed:
Fix link to open bug reports

Commit:
Merge branch 'runephilosof-abtion-patch-1'

vladdoster committed:
(docs): update `project-security/verifying-signatures.md`

- use active voice where applicable
- add articles
- add `shell_session` to code fences to match document style
- reword phrases to increase readability & reduce verbiage

Commit:
Copyedit; wrap text

Commit:
Merge branch 'vladdoster-patch-1'

Commit:
Remove Fedora 33 from supported templates (EOL)

Commit:
Update U2F proxy doc when having non standard sys-usb qubes name

marmarek committed:
schedule: there will be 4.1.0-rc3

Commit:
Merge branch 'fix_u2f_doc' of github.com:tt-why/qubes-doc into tt-why-fix_u2f_doc

Commit:
Merge branch 'tt-why-fix_u2f_doc'

unman committed:
Update qvm-prefs output to 4.0 format

Commit:
Add Debian 11 support for Qubes 4.0

QubesOS/qubes-issues#6865

Commit:
Merge branch '6911' of github.com:unman/qubes-doc into unman-6911

Commit:
Merge branch 'unman-6911'

smoothie-on-the-rocks committed:
Add fedora details to convert VM to Qube section

smoothie-on-the-rocks committed:
Change inline commands to block for easy selection

This slight change in format allow to select the command by triple clicking it. Currently, it selects the whole line. Also, when viewed from github, the block can be copied by clicking in the link on the right side.

Commit:
Merge branch 'smoothie-on-the-rocks-patch-2'

Commit:
Update URLs

https://forum.qubes-os.org/t/8019/

deeplow committed:
note possibility to add rpc services in app qube

The technical possibility can be found here: https://github.com/QubesOS/qubes-issues/issues/3003

Commit:
Merge branch 'patch-1' of github.com:smoothie-on-the-rocks/qubes-doc into smoothie-on-the-rocks-patch-1

Commit:
Merge branch 'smoothie-on-the-rocks-patch-1'

Commit:
Merge branch 'patch-28' of github.com:deeplow/qubes-doc into deeplow-patch-28

Commit:
Merge branch 'deeplow-patch-28'

deeplow committed:
add documentation for openqa_investigator script

Commit:
Fix typo

Commit:
Merge branch 'deeplow-openqa-update'

siliconninja committed:
Fix git commit code signing command

marmarek committed:
Update R4.1 release schedule

Move rc4 a bit earlier - there are only few bugs, and they already have
fixes developed, so give more time for testing instead.

test90 committed:
merge remote origin

Commit:
Merge branch 'pgp-instruction-improvements' of github.com:siliconninja/qubes-doc into siliconninja-pgp-instruction-improvements

Commit:
Merge branch 'siliconninja-pgp-instruction-improvements'

test90 committed:
Merge branch 'master' of https://github.com/QubesOS/qubes-doc
